### PR TITLE
transform log custom action parameters

### PR DESCRIPTION
**Pull request by jafowler**
log custom fields action parameters are not being transformed correctly to match the schema.
Before:
```
resource "cloudflare_ruleset" {
  kind    = "zone"
  name    = "default"
  phase   = "http_log_custom_fields"
  zone_id = ""
  rules {
    action      = "log_custom_field"
    description = "Set Logpush custom fields for HTTP requests"
    enabled     = true
    expression  = "true"
    action_parameters {
      cookie_fields = [
        {
          name = "asdf"
        },
        {
          name = "zxcv"
        }
      ]
    }
  }
}
```
After:
```
resource "cloudflare_ruleset" {
  kind    = "zone"
  name    = "default"
  phase   = "http_log_custom_fields"
  zone_id = ""
  rules {
    action      = "log_custom_field"
    description = "Set Logpush custom fields for HTTP requests"
    enabled     = true
    expression  = "true"
    action_parameters {
      cookie_fields = ["asdf", "zxcv"]
    }
  }
}
```

I purposefully did this specific to log custom fields. If more action_parameters start returning a list of values i think this can be transitioned to handle that. The current assumption we're making is that each of these elements will have a "name" key.

### CODE REVIEW

File: internal/app/cf-terraforming/cmd/generate.go [hunk 1]
```diff
@@ -724,6 +727,19 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 									}
 								}
 							}
+							// check for log custom fields that need to be transformed
+							for _, logCustomFields := range logCustomFieldsTransform {
```
Comment (manatarms) [inline internal/app/cf-terraforming/cmd/generate.go:731]: Can we write a test for this?

Comment (jacobbednarz) [reply internal/app/cf-terraforming/cmd/generate.go:731]: definitely a good idea. checkout https://github.com/cloudflare/cf-terraforming/blob/master/testdata/cloudflare/cloudflare_ruleset_zone_ddos_l7.yaml and https://github.com/cloudflare/cf-terraforming/tree/master/testdata/terraform/cloudflare_ruleset_zone_ddos_l7 for putting one together.

Comment (jafowler) [reply internal/app/cf-terraforming/cmd/generate.go:731]: I'll work on getting some tests added. Going to need to get the acceptance test zone entitled to use the new phase.

Comment (jacobbednarz) [reply internal/app/cf-terraforming/cmd/generate.go:731]: these tests are all mocked so no real HTTP requests are made. if you know what the payload is, you can throw it into the file.

**Pull request merged**